icu4j/main/tests/core/src/com/ibm/icu/dev/test/lang/UCharacterCaseTest.java

```diff
@@ -467,6 +470,72 @@ public void TestDutchTitle() {
         assertEquals("Dutch titlecase check in Dutch with nolowercase option",
                 "IJssel Igloo IJMUIdEN IPoD IJenough",
                 UCharacter.toTitleCase(LOC_DUTCH, "ijssel igloo IjMUIdEN iPoD ijenough", iter, options));
+
+        // Accented IJ testing
+
+        String[][] dutchIJCasesData = {
+                // input,           expected,         expOnlyChanged
+                {"ij",              "IJ",             "IJ"},
+                {"IJ",              "IJ",             ""},
+                {"íj́",              "ÍJ́",             "ÍJ"},
```

Comment:
When the input is the lowercase accented digraph (`"íj́"`), the omit-unchanged string omits the accent mark after the capital 'J'. (`"ÍJ"`, whereas full titlecase string is `"ÍJ́"`) That means the changed region splits a grapheme cluster, right? If so, is that expected behavior for users?